plcc/src/plcc/Settings.java

```diff
@@ -512,7 +513,7 @@ public static Boolean setDefaults() {
         defSet("plcc_I_number_of_graphlets", "30", "The length of the graphlet vector in the database (the PostgreSQL SQL array). This is the number of graphlets used to compute similarity.");
         
         // rna
-        defSet("plcc_B_include_rna", "false", "Whether RNA should be parsed and included in graph formalism and visualisation (WIP)");
+        defSet("plcc_B_include_rna", "true", "Whether RNA should be parsed and included in graph formalism and visualisation (WIP)");
```

Comment:
As long as this is work in progress (as marked by the description of the setting itself) I would not make it a default setting.